cli/azd/pkg/executil/executil_test.go

```diff
@@ -161,3 +161,54 @@ func TestRunEnrichError(t *testing.T) {
 	// themselves in checking the ExitCode.
 	require.EqualError(t, err, fmt.Sprintf("%s: exit status 2", res.String()))
 }
+
+func TestRedactSensitiveData(t *testing.T) {
```

Comment:
Just pointing out you could write the three test cases here as a "table-driven style" test, where you set up all your expected input/output before-hand, and write a for-loop that will execute each test matching input to output:

Here's a documented [example](https://gobyexample.com/testing).

Pseudocode related to your test case:

```golang
tests := []struct{
  scenario string,
  input string,
  expected string) {
  {scenario: "Basic",
   input: `"accessToken": "eyJ0eX",
"expiresOn": "2022-08-11 10:33:39.000000",
"subscription": "2cd61",
"tenant": "72f988bf",
"tokenType": "Bearer"
}`,
  expected: `"accessToken": "redact to prevent sensitive data",
"expiresOn": "2022-08-11 10:33:39.000000",
"subscription": "2cd61",
"tenant": "72f988bf",
"tokenType": "Bearer"
}`},
  {scenario: "NoReplacement,", ... },
  {scenario: "MultipleReplacement,", ... },
}

for _, test := range tests {
  t.Run(test.scenario, func() {
    actual := redactSensitiveData(test.input)
    require.Equal(t, test.expected, actual)
  })
}
```